pkg/actors/actors.go

```diff
@@ -1106,13 +1142,26 @@ func (a *actorsRuntime) CreateTimer(ctx context.Context, req *CreateTimerRequest
 					if err != nil {
 						log.Errorf("error invoking timer on actor %s: %s", actorKey, err)
 					}
-				} else {
+					if repeats > 0 {
+						cnt++
+					}
+				}
+				if !exists || (repeats > 0 && cnt >= repeats) {
```

Comment:
So is `cnt` only incremented in the case where the actor doesn't exist? Don't we need to for successful invocations as well?